test/e2e/kubectl.go

```diff
@@ -125,7 +125,7 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 	framework.KubeDescribe("Update Demo", func() {
 		var updateDemoRoot, nautilusPath, kittenPath string
 		BeforeEach(func() {
-			updateDemoRoot = filepath.Join(framework.TestContext.RepoRoot, "docs/user-guide/update-demo")
+			updateDemoRoot = filepath.Join(framework.TestContext.GeneratedAssetsDir, framework.GENERATED_ASSETS, "docs/user-guide/update-demo")
```

Comment:
I'm slightly confused, how are the examples embedded?  Don't they depend on this asset directory now?   

I was under the impression that you were going to generate static data that would be embedded into the code so the e2e's don't rely on repo-root at all.?.? But it looks like this trades 1 thing for another, but we still have the same problem.